### PR TITLE
Use page-specific titles

### PR DESCRIPTION
**Pull request by jonc125**
To make the browser back button menu more informative!

### CODE REVIEW

**Comment by helenst** [comment]
Coverage shouldn't be complaining since this only affects template files. Might be because it needs rebasing... trying that now.

**Comment by jonc125** [comment]
Nice! I don't quite understand why codecov complains some times - it seems even adding new fully covered code can sometimes decrease overall coverage?

File: weblab/templates/entities/entity_list.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% extends "base.html" %}
 {% load entities %}
 
+{% block title %}My {{ type }}s - {% endblock title %}
```
Comment (helenst) [inline weblab/templates/entities/entity_list.html]: Title uses "My" but page heading uses "Your". We should pick one :) The interface isn't very consistent on this front anyway (has both my files / your files) - carried over from v1 I think so might need some thought as to which we use.

Comment (jonc125) [reply weblab/templates/entities/entity_list.html]: Good point! I think 'your' is probably better. Do @mirams @MichaelClerx have an opinion?

Comment (MichaelClerx) [reply weblab/templates/entities/entity_list.html]: I prefer 'your' !
Alternatively: <user name>'s

**Pull request merged**